packages/block-library/src/comment-template/edit.js

```diff
@@ -23,11 +28,89 @@ const TEMPLATE = [
 	[ 'core/comment-edit-link' ],
 ];
 
-function CommentTemplateInnerBlocks() {
+function CommentTemplateInnerBlocks( {
+	comments,
+	activeBlockContext,
+	setActiveBlockContext,
+	blocks,
+} ) {
 	const innerBlocksProps = useInnerBlocksProps( {}, { template: TEMPLATE } );
-	return <li { ...innerBlocksProps } />;
+	return (
+		<li { ...innerBlocksProps }>
+			{ innerBlocksProps.children }
```

Comment:
I hope I'm using it correctly here! Took me a while to figure out that I should pass the `children` manually here if `<li { ...innerBlocksProps }>` itself has children.

Reply:
I would destruct the result of the hook:

```js

const { children, …innerBlocksProps } = useInnerBlocksProps( {}, { template: TEMPLATE } );
```

then everything should read better. When you explicitly set the children it overrides the one passed as prop so technically it should work the same.

Reply:
oh, yeah, that's a bit clearer. 👍